src/core_codemods/fix_deprecated_abstractproperty.py

```diff
@@ -19,6 +19,11 @@ class FixDeprecatedAbstractproperty(BaseCodemod, NameResolutionMixin):
     def leave_Decorator(
         self, original_node: cst.Decorator, updated_node: cst.Decorator
     ):
+        if not self.filter_by_path_includes_or_excludes(
+            self.node_position(original_node)
+        ):
+            return original_node
```

Comment:
Sorry for not realizing this before, but we should return `updated_node` here just in case some node within the `original_node` subtree was changed. Otherwise we may revert those previous changes.

I don't think it will affect the current codemods that contains this snippet, but this will probably be copy/pasted a lot, so it should include the "safe" version.

Reply:
I noticed we don't always return updated_node like the docs indicate we should. I kinda wanted this to read "if we are excluding this node bc of line, don't change". so returning original_node reads "no change happened" but  it's true that's not always the case.